export const singIn = () => {
  try {
  } catch (err) {
    throw err;
  }
};
